tutorials/en_us/JARS/FINDING.md

```diff
@@ -6,7 +6,7 @@ Here we will go through how to pick what server jar to use. A server jar contain
 
 The first step to finding the right server jar for you is to know what kind of server you want to run, is it survival, prisons, minigames and what Minecraft version is it? We will show you some basic ones then you can look at (link to the page of server jars) for more in-depth explanations etc.
 
-### For MC 1.6.X
+### For MC 1.16.X
```

Comment:
Good catch!

Reply:
actually it was @Silverarmor who found it, but noone ever changed it so i made the pull request.